docs/excel/custom-functions-overview.md

```diff
@@ -72,32 +72,32 @@ Excel.run(function(ctx) {
+-   `call`: Defines the JavaScript function to call (for example, `add42`). The name of the JavaScript function does not need to match the name that you register in Excel.
+-   `description`: The description appears in the autocomplete menu in Excel.
+-   `helpUrl`: When the user requests help for a function, Excel opens a task pane and displays the web page found at this URL.
+-   `result`: Defines the type of information returned by the function to Excel.
 
-    -   resultType: Your function can return either a number or a string. For more information see &lt;&lt;LINK&gt;&gt;
+    -   `resultType`: Your function can return either a `"string"` or a `"number"` (also used for dates and currencies). For more information see &lt;&lt;LINK&gt;&gt;.
 
-    -   resultDimensionality: Your function can return either a single value or a matrix of values. When returning a matrix of values, your function returns an array, where each array element is another array of values. For more information, see &lt;&lt;LINK&gt;&gt;. The following example returns a 3-row, 2-column matrix of values from a custom function.
+    -   resultDimensionality: Your function can return either a single (`"scalar"`) value or a `"matrix"` of values. When returning a matrix of values, your function returns an array, where each array element is another array that represents a row of values. For more information, see &lt;&lt;LINK&gt;&gt;. The following example returns a 3-row, 2-column matrix of values from a custom function.
 
 ```js
-return [[1,1],[2,2],[3,3]];
+return [["first","row"],["second","row"],["third","row"]];
 ```
 
--   Your custom function may take arguments as input. The arguments passed to your custom function are specified in the *parameters* property. If you declare more than 1 parameter, ensure the order of the parameter definition matches your JavaScript function. The parameter property arguments are described as follows:
+-   Your custom function may take arguments as input. The arguments passed to your custom function are specified in the *parameters* property. The order of the parameters in the definition must match the order in the JavaScript function. For each parameter, define these properties:
 
-    -   name: The string displayed in Excel representing the parameter name.
-    -   description: A description of the parameter.
-    -   valueType: A number or string, that is similar to the resultType described earlier.
-    -   valueDimensionality: A single value or matrix of values, that is similar to the resultDimensionality described previously. Matrix-type parameters allow the user to select ranges larger than a single cell.
+    -   `name`: The string displayed in Excel to represent the parameter.
+    -   `description`: The string displayed for more information about the parameter.
+    -   `valueType`: A `"number"` or `"string"`, similar to the resultType property described earlier.
```

Comment:
Please remove the quotes from string and number. Thanks.

Reply:
same reason as above